Userland/Libraries/LibGfx/AffineTransform.cpp

```diff
@@ -18,7 +18,7 @@ bool AffineTransform::is_identity() const
 static float hypotenuse(float x, float y)
```

Comment:
What does this function even do now?

Reply:
calculates the hypotenuse given the legs, I guess....